shopify_integration/shopify_integration/doctype/shopify_settings/shopify_settings.json

```diff
@@ -309,7 +309,7 @@
    "reqd": 1
   },
   {
-   "depends_on": "eval:doc.app_type==\"Public\"",
+   "depends_on": "eval:(in_list([\"Public\", \"Custom (OAuth)\"], doc.app_type))",
```

Comment:
Is there a "Custom" app that's not Oauth? Also is it called private or custom?

Reply:
Yep, it's called Custom, and it uses basic auth

https://shopify.dev/apps/getting-started/app-types